### ISSUE TITLE
Gutenberg editor text block classic block

### ISSUE THREAD

**Issue by NoelOdanen** [WordPress/gutenberg#15143]
**Describe the bug**

**To reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
- Please add the version of Gutenberg you are using in the description.
- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.

**Comment by NoelOdanen** [comment]
The gutenberg version I have was different & excellent. Today all the pages changed to the "text block" or "classic block" (sorry, Im translating from spanish). And it enabled the possibility to edit the yoast SEO. Can you give me the Gutenberg back?

**Comment by ajitbohra** [comment]
Hi there,

Thanks for reporting your issue.  We need more information before we can assist with reproducing and potentially fixing.  Can you please follow the [bug report template](https://github.com/WordPress/gutenberg/issues/new?template=Bug_report.md) when crafting your issue.  The questions there assist in providing the information needed to help with reproducing and addressing reported issues.

There is not enough information in your report for this to be actionable so I'm going to close this for now but feel free to open a new issue following the template and with more details.
